src/context/Queue.ts

```diff
@@ -87,6 +88,7 @@ export class Queue {
 
     addOp(operation: OperationFn, options = {}) {
         this.operations.push({
+            id: Date.now(),
```

Comment:
Using `Date.now()` is not safe for identifiers. For example:

```
const x = Date.now()
const y = Date.now()
console.log(x, y)
```

On my machine this logs `1656987102933 1656987102933`. Instead consider using `nanoid`, we already have it as part of the unirep package.